### PR TITLE
Create update-v1-to-v2.md

### PR DESCRIPTION
**Pull request by dirkmc**
Add instructions to migrate from v1 to v2

### CODE REVIEW

File: tutorials/update-v1-to-v2.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+```
+./migrate-lid yugabyte \
+    --connect-string yugabyte://<yugabyte-ip> \
+    --hosts <yugabyte-ip> \
+    dagstore
+```
+
+It will output a progress bar, and also a log file with detailed migration information at `migrate-yugabyte.log`
+
+#### 5. Update boostd config
```
Comment (davidd8) [inline tutorials/update-v1-to-v2.md:61]: Can we be clear about which config for boostd is being updated: the original directory, or the tmp dir?

File: tutorials/update-v1-to-v2.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+For example:
+```
+[LocalIndexDirectory]
+  [LocalIndexDirectory.Yugabyte]
+    ConnectString = "yugabyte://127.0.0.1"
+    Hosts = "127.0.0.1"
+```
+
+#### 6. Install boostd v2
```
Comment (davidd8) [inline tutorials/update-v1-to-v2.md:73]: Again is this being installed in the new /tmp dir or in the original dir, does it matter?

File: tutorials/update-v1-to-v2.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+make install
+make booster-http
+make booster-bitswap
+```
+
+Note that in v2 booster-http and booster-bitswap take slightly different parameters (see below).
+
+#### 7. Stop boostd, booster-http and booster-bitswap
+
+Note: You need to stop boostd before migrating piece info data.
```
Comment (davidd8) [inline tutorials/update-v1-to-v2.md:85]: Add the commands here for stopping each service.

File: tutorials/update-v1-to-v2.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+#### 6. Install boostd v2
+
+```
+make install
+make booster-http
+make booster-bitswap
+```
+
+Note that in v2 booster-http and booster-bitswap take slightly different parameters (see below).
```
Comment (davidd8) [inline tutorials/update-v1-to-v2.md:81]: where are the parameters?

File: tutorials/update-v1-to-v2.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+#### 7. Stop boostd, booster-http and booster-bitswap
+
+Note: You need to stop boostd before migrating piece info data.
+
+#### 8. Migrate piece info data (information about which sector each deal is stored in)
+
+This should take no more than a few minutes.
+```
+./migrate-lid yugabyte \
```
Comment (davidd8) [inline tutorials/update-v1-to-v2.md:91]: Again, is everything migrating from the /tmp dir?

File: tutorials/update-v1-to-v2.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+#### 9. Start v2 of boostd, booster-http and booster-bitswap
+
+Note that booster-http and booster-bitswap take slightly different parameters:
+- The parameter `api-boost` is removed
+- There is a new parameter `api-lid` that points to the LID service
+- By default the LID service is started with boostd, on the same machine, at port 8042
+- The connection must be made over a web socket (use `ws` in the url instead of `http`)
+- eg `ws://localhost:8042`
+
```
Comment (davidd8) [inline tutorials/update-v1-to-v2.md:105]: Show the success in the command prompt! As a TODO, add a what's next in the migration step, are you done, or is there more you *can* do?

Comment (davidd8) [reply tutorials/update-v1-to-v2.md:105]: When do you delete the /tmp dir?

File: tutorials/update-v1-to-v2.md (hunk 1 above)
Comment (dirkmc) [reply tutorials/update-v1-to-v2.md:61]: The config file lives in the boost repo, by default in the home directory.
I changed to `<repo>/boost.config` to make this clearer.

File: tutorials/update-v1-to-v2.md (hunk 2 above)
Comment (dirkmc) [reply tutorials/update-v1-to-v2.md:73]: Added `cd /tmp/boostv2` to make it clearer

File: tutorials/update-v1-to-v2.md (hunk 3 above)
Comment (dirkmc) [reply tutorials/update-v1-to-v2.md:85]: There isn't really a command, you just send a kill signal. The way to do this differs depending on how the SP has set things up (eg using systemctl)

File: tutorials/update-v1-to-v2.md (hunk 4 above)
Comment (dirkmc) [reply tutorials/update-v1-to-v2.md:81]: At the bottom of the tutorial

File: tutorials/update-v1-to-v2.md (hunk 5 above)
Comment (dirkmc) [reply tutorials/update-v1-to-v2.md:91]: The command is run from the directory that was created above. If people copy blindly copy and paste the commands from each step it should work.

File: tutorials/update-v1-to-v2.md (hunk 6 above)
Comment (dirkmc) [reply tutorials/update-v1-to-v2.md:105]: /tmp gets automatically cleaned up by the OS

**Comment by dirkmc** [comment]
Suggestions addressed at [update boost v1 to v2 tutorial](https://github.com/filecoin-project/boost-docs/pull/45#top)

**Pull request merged**